packit_service/worker/checker/distgit.py

```diff
@@ -34,7 +34,7 @@ def contains_specfile_change(self):
             return True
 
         pr_id = self.pull_request.id
-        diff = self.project.get_pr_files_diff(pr_id) or {}
+        diff = self.project.get_pr_files_diff(pr_id, retries=3) or {}
```

Comment:
I would be explicit about the delay, so it is obvious we can adjust it here:

```suggestion
        # Pagure API tends to return ENOPRSTATS error when a pull request is transitioning
        # from open to merged state, give it some extra time
        diff = self.project.get_pr_files_diff(pr_id, retries=3, wait_seconds=3) or {}
```

Reply:
Do we want to handle the `PagureAPIException` exception here?

Reply:
I would let this as you wrote it 🙂